gulpfile.js

```diff
@@ -0,0 +1,32 @@
+var gulp = require('gulp');
+var zip = require('gulp-zip');
+var del = require('del');
+var runSequence = require('run-sequence');
+var path = require('path');
+var buildDest = path.resolve("./build");
+
+var srcFiles = path.resolve('./src/**/*.*');
```

Comment:
should perhaps take another look at this, i didn't know you could pass a glob to path.resolve, does it result in a glob with an absolute path?

also, both this and buildDest(buildPath/buildDestination?) could just be relative